From e77b761cd5915ab2384aa60c4862368199e40551 Mon Sep 17 00:00:00 2001
From: Harsh Shandilya <msfjarvis@gmail.com>
Date: Sun, 16 Jul 2017 21:18:59 +0530
Subject: [PATCH 14/14] interfacer: Allow interfacer to find content_service

https://substratum.review/#/c/420/ implements a ContentObserver in
interfacer which requires interfacer to be able to find the content_service.

Change-Id: I1d8cabd9848807ea4dfafcf7123478da834ef5a5
---
 interfacer.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/interfacer.te b/interfacer.te
index a5ba1d7..fc6921a 100644
--- a/interfacer.te
+++ b/interfacer.te
@@ -57,6 +57,7 @@ allow interfacer media_rw_data_file:file rw_file_perms;
 # Services
 allow interfacer activity_service:service_manager find;
 allow interfacer connectivity_service:service_manager find;
+allow interfacer content_service:service_manager find;
 allow interfacer display_service:service_manager find;
 allow interfacer mount_service:service_manager find;
 allow interfacer network_management_service:service_manager find;
-- 
2.13.3

